package-lock.json

```diff
@@ -3878,6 +3881,15 @@
       "integrity": "sha512-/+CRPXpBDpo2RK9C68N3b2cOvO0Cf5B9aPijHsoDQTHivnGSObdOF2BRQOYjojWTDy6nQvMjmqRXIxH55VjxxA==",
       "dev": true
     },
+    "node_modules/@types/axios": {
+      "version": "0.14.0",
+      "resolved": "https://registry.npmjs.org/@types/axios/-/axios-0.14.0.tgz",
+      "integrity": "sha1-7CMA++fX3d1+udOr+HmZlkyvzkY=",
+      "deprecated": "This is a stub types definition for axios (https://github.com/mzabriskie/axios). axios provides its own type definitions, so you don't need @types/axios installed!",
+      "dependencies": {
+        "axios": "*"
+      }
+    },
```

Comment:
I don't know why add `@types/axios` to `package-lock.json`. 🤔

Reply:
[`@types/axios` has been in `package.json`](https://github.com/planetarium/9c-launcher/blob/1b8acf4988563b639aa9b886f101804258666be1/package.json#L62) but the `package-lock.json` seems not updated at that time. And `@types/axios` should be removed because `@types/axios` was deprecated and `axios` has supported its type itself.

For `node_modules/@types/axios`, According to [GitHub (npm) blog, there was a patch related to *package-lock.json* since npm v7](https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/). So IMO it seems you are using npm v7 and the `node_modules/@types/axios` were added also.

Reply:
It was added at 52a215d3d3e1dd340e033da6ef0d875a1a927c61. @riemannulus is it intended change? I think is not related to fixing tsconfig path.

Reply:
That's not intended. It is a just mistake.

Reply:
Can I remove `@types/axios` in this PR? (or create new PR?)

Reply:
In my thought, it is better to remove in new PR.